zerver/lib/validator.py

```diff
@@ -72,10 +82,9 @@ def f(var_name, val):
     return f
 
 def check_dict(required_keys):
-    # required_keys is a list of tuples of
-    # key_name/validator
-
```

Comment:
I might replace this with an example in a docstring, since this is kinda confusing to figure out how to use if you're not looking at an example...